cmd/delete.go

```diff
@@ -17,17 +16,24 @@ var deleteCmd = &cobra.Command{
 	RunE:  delete,
 }
 
+var preserveKey bool
+
 func init() {
+	deleteCmd.Flags().BoolVar(&preserveKey, "preserve-key", false, "Prevent normalization of the provided key in order to delete any keys that may not have been previously normalized on write.")
```

Comment:
I'm not so sure how I feel about this being `preserve-key`.

On one hand, it makes it easy to quickly enable this behavior by doing
```
chamber delete --preserve-key
```
keeping custom behavior the "falsy" path.

On the other hand, it can be confusing for a few reasons:
* on a `delete` command, this could come across as "delete the value in the store but keep the key defined in the store" (essentially just nullify the value)
* it differs in naming/convention/standard from the `--normalize-key` added on the `import` command, so there isn't consistency for what is effectively the same behavior
  * Note, it felt bad to have this be `normalize-key` as well, but with a default of `true`

I'm open to any and all suggestions for making this better

Reply:
How do you feel about just reusing `--normalize-key` for `delete` as well?

Reply:
@mckern definitely agree with you, but removing feels like a breaking change in behavior we would have to account for in a major version bump, right?

Reply:
Just noticed your edit - happy to move to `--no-case` over `--preserve-key` 👍

Reply:
context: we're digging into how AWS handles same keys with different casings to understand impact of deleting a key that is `MY_SECRET_VALUE` if the provided key is `my_secret_value`

Reply:
Resolved offline - AWS allows for case sensitive unique keys, so `MY_SECRET_VALUE` and `my_secret_value` are both treated as unique. Changing `--preserve-key` to `--exact-key` to make it clearer in the command line